### ISSUE TITLE
[Suggestion] Increase/Decrease a field value with the mouse wheel or the arrow keys

### ISSUE THREAD

**Issue by RgehanProjects** [UnderwaterApps/overlap2d#43]
Photoshop does that, and it's quite useful.

**Comment by kutoman** [comment]
I think I will make a pull request for that

**Comment by RgehanProjects** [comment]
This is going to be useful

**Comment by azakhary** [comment]
Good point.

Btw, did you knew that you can rotate an item while dragging if you scroll mouse wheel?

**Comment by RgehanProjects** [comment]
Yes, I knew that, but I think it won't conflict, because we're not usually editing textfields while dragging objects :)

**Comment by azakhary** [comment]
yeah sure, it wouldn't.

**Comment by DarthJDG** [comment]
In my opinion the scrollwheel rotate while dragging is quite uncomfortable. This would definitely be a useful feature. I would love to have an additional on-screen anchor to drag for rotation, or some modifier key to press and hold.
